### PR TITLE
wildmidi: update to 0.4.4

### PR DESCRIPTION
**Pull request by Oreo639**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Oreo639** [comment]
Save as midi seems to still be broken. I'll have to make a new patch.

File: srcpkgs/wildmidi/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
 license="GPL-3.0-or-later, LGPL-3.0-only"
 homepage="http://www.mindwerks.net/projects/wildmidi"
```
Comment (ericonr) [inline srcpkgs/wildmidi/template:14]: Would you mind switching to `https` here?

Comment (Oreo639) [reply srcpkgs/wildmidi/template]: Sure.

**Comment by ericonr** [comment]
Ugh, the release notes have 

> Fixed broken file name handling for 'save as midi' command during playback.

but don't show the fix. How is it broken? I can help with the patch, if you need it (and please look into upstreaming it as well).

**Comment by Oreo639** [comment]
The issue here is a different issue than the issue before. The issue before is explained here:
https://github.com/Mindwerks/wildmidi/commit/ef50330257b60049b357e2729d88ddda82f4e5f6
The issue here is due to this:
https://github.com/Mindwerks/wildmidi/blob/master/src/wildmidi.c#L1999-L2000

**Comment by ericonr** [comment]
Could you please open an issue/PR upstream so we can get thoughts on whatever patch we decide to ship? Thanks for the info.

**Comment by Oreo639** [comment]
Already did, but thanks for asking.
https://github.com/Mindwerks/wildmidi/issues/227/

**Comment by ericonr** [comment]
Great, thanks :)

**Pull request merged**